### ISSUE TITLE
Glances installation via install.sh script fails on VPS

### ISSUE THREAD

**Issue by thorian93** [nicolargo/glancesautoinstall#8]
#### Description
When installing glances via the install.sh script on a VPS hosted by Strato the installation fails with the following messages:
Server 1:
`Detected system: strato`
Server 2:
`Detected system: plesk`

The scripts seems to be unable to use the `lsb_release` tool on the servers although it is installed. When using the workaround:
`find /etc -type f -maxdepth 1 \( ! -wholename /etc/os-release ! -wholename /etc/lsb-release -wholename /etc/\*release -o -wholename /etc/\*version \) 2> /dev/null`
it gets the following outputs:
`/etc/strato-release /etc/debian_version`
and
`/etc/plesk-release /etc/debian_version /etc/strato-release`

which result in the error that this is a unsupported system.
#### Servers
Ubuntu 14.04 LTS 64bit
Ubuntu 14.04 LTS 64bit + Plesk 12.5

As far as I can see that this issue is related to the way the distribution detection is done.
Maybe the script simply needs to be expanded with the values I stated above. I would create a pull request if that is the case. If further development would be needed, then I personally can't help.

If further information is needed I will provide it.

I appreciate any support!

**Comment by thorian93** [comment]
@nicolargo: You have no contribution guide so I'll just ask straight ahead:  
Should I modify the install-develop.sh script or the install.sh script itself?

**Comment by nicolargo** [comment]
Both !

**Comment by thorian93** [comment]
I looked into the code and as far as I see commit 8190643 added the check with lsb_release and a workaround if it is not available. Unfortunately @tomauty either made a mistake or I miss something.  
Here's why:
Instead of checking for `lsb_release` he checked for `lsb_releaseX` with a upper case <b>X</b> at the end.
I don't know much about Arch and may miss something but if it's a typo I corrected it in pull request #9.
